samples/python/video.py

```diff
@@ -156,73 +164,80 @@ def render(self, dst):
 classes = dict(chess=Chess, book=Book, cube=Cube)
 
 presets = dict(
-    empty = 'synth:',
-    lena = 'synth:bg=lena.jpg:noise=0.1',
-    chess = 'synth:class=chess:bg=lena.jpg:noise=0.1:size=640x480',
-    book = 'synth:class=book:bg=graf1.png:noise=0.1:size=640x480',
-    cube = 'synth:class=cube:bg=pca_test1.jpg:noise=0.0:size=640x480'
+    empty="synth:",
+    lena="synth:bg=lena.jpg:noise=0.1",
+    chess="synth:class=chess:bg=lena.jpg:noise=0.1:size=640x480",
+    book="synth:class=book:bg=graf1.png:noise=0.1:size=640x480",
+    cube="synth:class=cube:bg=pca_test1.jpg:noise=0.0:size=640x480",
 )
 
 
-def create_capture(source = 0, fallback = presets['chess']):
-    '''source: <int> or '<int>|<filename>|synth [:<param_name>=<value> [:...]]'
-    '''
+def create_capture(source=0, fallback=presets["chess"]):
+    """source: <int> or '<int>|<filename>|synth [:<param_name>=<value> [:...]]'"""
     source = str(source).strip()
 
     # Win32: handle drive letter ('c:', ...)
-    source = re.sub(r'(^|=)([a-zA-Z]):([/\\a-zA-Z0-9])', r'\1?disk\2?\3', source)
-    chunks = source.split(':')
-    chunks = [re.sub(r'\?disk([a-zA-Z])\?', r'\1:', s) for s in chunks]
+    source = re.sub(r"(^|=)([a-zA-Z]):([/\\a-zA-Z0-9])", r"\1?disk\2?\3", source)
+    chunks = source.split(":")
+    chunks = [re.sub(r"\?disk([a-zA-Z])\?", r"\1:", s) for s in chunks]
 
     source = chunks[0]
-    try: source = int(source)
-    except ValueError: pass
-    params = dict( s.split('=') for s in chunks[1:] )
+    try:
+        source = int(source)
+    except ValueError:
+        pass
+    params = dict(s.split("=") for s in chunks[1:])
 
     cap = None
-    if source == 'synth':
-        Class = classes.get(params.get('class', None), VideoSynthBase)
-        try: cap = Class(**params)
-        except: pass
+    if source == "synth":
+        Class = classes.get(params.get("class", None), VideoSynthBase)
+        try:
+            cap = Class(**params)
+        except:
+            pass
     else:
         cap = cv.VideoCapture(source)
-        if 'size' in params:
-            w, h = map(int, params['size'].split('x'))
+        if "size" in params:
+            w, h = map(int, params["size"].split("x"))
             cap.set(cv.CAP_PROP_FRAME_WIDTH, w)
             cap.set(cv.CAP_PROP_FRAME_HEIGHT, h)
     if cap is None or not cap.isOpened():
-        print('Warning: unable to open video source: ', source)
+        print("Warning: unable to open video source: ", source)
         if fallback is not None:
             return create_capture(fallback, None)
     return cap
 
-if __name__ == '__main__':
+
+if __name__ == "__main__":
```

Comment:
Where did you get this PEP8 configuration?

Just take a look on Python official documentation: https://docs.python.org/3/library/__main__.html